index.html

```diff
@@ -472,6 +472,7 @@
       object contentType;
       SemanticType[] semanticTypes;
       any value;
+      boolean readOnly = false;
```

Comment:
I'd suggest `boolean writeable = true;` instead (to be consistent with JS property descriptors).